AmplifyPlugins/API/Sources/AWSAPIPlugin/Operation/AWSGraphQLSubscriptionTaskRunner.swift

```diff
@@ -113,8 +114,8 @@ public class AWSGraphQLSubscriptionTaskRunner<R: Decodable>: InternalTaskRunner,
             self.subscription = try await appSyncClient?.subscribe(
                 id: subscriptionId,
                 query: encodeRequest(query: request.document, variables: request.variables)
-            ).sink(receiveValue: { [weak self] event in
-                self?.onAsyncSubscriptionEvent(event: event)
+            ).sink(receiveValue: { event in
+                self.onAsyncSubscriptionEvent(event: event)
```

Comment:
why do we remove `weak self` from this?

Reply:
sorry, cannot remember, will add it back.